profiles/arch/hppa/use.stable.mask

```diff
@@ -4,6 +4,12 @@
 # This file requires eapi 5 or later. New entries go on top.
 # Please use the same syntax as in use.mask
 
+# Andreas Sturmlechner <asturm@gentoo.org> (29 Oct 2017)
+# Mask qt5 for stabilisation timeout, qt4 b/c qtchooser...
```

Comment:
wait, what's the problem with qtchooser?

Reply:
repoman complained after dropping to ~hppa and maybe I didn't look close enough.

Reply:
you can just mask `USE=test` for qtchooser

Reply:
should I just fix the comment? it's not like we do not end up de-stabilising those packages anyway if qt4 support is not able to upgrade to qt5 support.

Reply:
well my point is that you didn't need to touch qt4 stuff in this PR, but since you already did the work, I have no preference either way. We'll have to remove those qt4 revdeps eventually anyway...